corehq/apps/app_manager/static/app_manager/js/vellum/manifest.txt

```diff
@@ -259,7 +259,7 @@
 │  └─ parse-filepath@^1.0.1
 ├─ flagged-respawn@1.0.1
 ├─ flat@5.0.2
-├─ follow-redirects@1.14.8
+├─ follow-redirects@1.15.5
```

Comment:
These changes seem to come from [this Vellum PR](https://github.com/dimagi/Vellum/pull/1104) and isn't directly related to the case micro image PR.